### PR TITLE
Journalføring papirsøknad logiskevedlegg advarsel

### PR DESCRIPTION
**Pull request by blommish**
Indirekt koblet til https://favro.com/organization/98c34fb974ce445eac854de0/a64c6aad9b0d61ef6c0290bd?card=Tea-9302

image

### CODE REVIEW

File: src/frontend/Komponenter/Journalforing/VelgUstrukturertDokumentasjonType.tsx [hunk 1]
```diff
@@ -21,6 +31,22 @@ const ustrukturertTypeTilTekst: Record<UstrukturertDokumentasjonType, string> =
     ETTERSENDNING: 'Ettersendning',
 };
 
+const advarselPapirsøknad = (
```
Comment (eirarset) [inline src/frontend/Komponenter/Journalforing/VelgUstrukturertDokumentasjonType.tsx:34]: småpirk
```suggestion
const AdvarselPapirsøknad = (
```

File: src/frontend/Komponenter/Journalforing/VelgUstrukturertDokumentasjonType.tsx [hunk 2]
```diff
@@ -76,9 +102,7 @@ const VelgUstrukturertDokumentasjonType: React.FC<{
                             ))}
                         </VelgUstrukturertDokumentasjonTypeSelect>
                         {ustrukturertDokumentasjonType ===
-                            UstrukturertDokumentasjonType.PAPIRSØKNAD && (
-                            <div>Årsak til behandling kommer settes til papirsøknad</div>
-                        )}
+                            UstrukturertDokumentasjonType.PAPIRSØKNAD && advarselPapirsøknad}
```
Comment (eirarset) [inline src/frontend/Komponenter/Journalforing/VelgUstrukturertDokumentasjonType.tsx:105]:
```suggestion
                            UstrukturertDokumentasjonType.PAPIRSØKNAD && AdvarselPapirsøknad}
```

**Review by eirarset — APPROVED** [review]
Ser bra ut 👏

File: src/frontend/Komponenter/Journalforing/VelgUstrukturertDokumentasjonType.tsx (hunk 1 above)
Comment (blommish) [reply src/frontend/Komponenter/Journalforing/VelgUstrukturertDokumentasjonType.tsx]: Takk! FIkset i [63d154e](https://github.com/navikt/familie-ef-sak-frontend/pull/1503/commits/63d154eb7004c97fd8116c158b17c51ec9d3a913)

**Pull request merged**